simulator/scheduler/plugin/plugins.go

```diff
@@ -21,45 +21,44 @@ import (
 // ResultStoreKey represents key name of plugins results on sharedstore.
 const ResultStoreKey = "PluginResultStoreKey"
 
-func NewRegistry(sharedStore storereflector.Reflector, cfg *schedulerConfig.KubeSchedulerConfiguration) (map[string]schedulerRuntime.PluginFactory, error) {
+func NewRegistry(sharedStore storereflector.Reflector, cfg *schedulerConfig.KubeSchedulerConfiguration, pluginExtenders map[string]PluginExtenderInitializer) (map[string]schedulerRuntime.PluginFactory, error) {
```

Comment:
add `pluginExtenders` to param so that the external scheduler can add pluginExtenders.